modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase

```diff
@@ -34,9 +34,43 @@ definition {
 	@description = "LPS-175645 - It's possible to add an object entry with a multi select picklist field of a picklist previously imported"
 	@priority = 4
 	test CanAddEntryWithMultiSelectPicklistImported {
+		task ("Given: A picklist imported and an object created") {
+			Picklist.gotoPicklists();
 
-		// TODO pending implementation
+			Picklist.importPicklist(
+				picklistFileName = "simple_picklist",
+				picklistName = "ImportedPicklist1");
+
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 1",
+				objectName = "CustomObject1",
+				pluralLabelName = "Custom Objects 1");
+		}
+
+		task ("When: User adds a multiselect picklist field and adds an object entry") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 1");
+
+			ObjectAdmin.goToFieldsTab();
 
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Field Picklist",
+				fieldPicklist = "ImportedPicklist1",
+				fieldType = "Multiselect Picklist");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject1");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject1");
+
+			ObjectAdmin.addObjectMultiPicklistFieldEntryViaUI(picklistOption = "Picklist Item 1");
```

Comment:
Since this test is for multiselect picklists, I think adding multiple picklist items at this step would be ideal.